Library/Homebrew/utils/github/api.rb

```diff
@@ -197,6 +197,8 @@ def self.credentials_error_message(response_headers, needed_scopes)
 
       scopes = response_headers["x-accepted-oauth-scopes"].to_s.split(", ")
       needed_scopes = Set.new(scopes || needed_scopes)
+      github_permission_link = GitHub.pat_blurb(needed_scopes)
```

Comment:
```suggestion
      github_permission_link = GitHub.pat_blurb(needed_scopes.to_a)
```

And move it down below the `return`.

Reply:
@reitermarkus changes done as requested